python/nav/models/apps.py

```diff
@@ -4,3 +4,6 @@
 class NavModelsConfig(AppConfig):
     name = 'nav.models'
     verbose_name = 'NAV models'
+
+    def ready(self):
+        import nav.models.signals  # noqa
```

Comment:
```suggestion
        import nav.models.signals  # noqa: F401
```
just to be specific which rule you're intentionally breaking